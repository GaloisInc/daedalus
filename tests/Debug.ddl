def Trace (s : [uint 8]) : {}
