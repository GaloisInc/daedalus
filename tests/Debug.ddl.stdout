module Debug
 
--- Imports:
 
--- Type defs:
 
--- Rules:
 
Debug.Trace (s : [uint 8]) : Grammar {}
 
Debug._Trace (s : [uint 8]) : Grammar {} =
  Debug.Trace s
